src/applications/ivc-champva/10-10D/helpers/utilities.js

```diff
@@ -40,36 +41,74 @@ export const applicantListSchema = (requireds, propertyList) => {
   };
 };
 
+// Extracting this to a function so there aren't a thousand identical
+// ternaries we have to change later
+export function sponsorWording(formData, isPosessive = true, cap = true) {
+  const retVal = isPosessive ? 'sponsor’s' : 'sponsor';
+  // Optionally capitalize first letter and return
+  return cap ? retVal.charAt(0).toUpperCase() + retVal.slice(1) : retVal;
+}
+
 /**
- * Gets the appropriate form key needed in the signature box so we can
- * corroborate who is signing the form.
- *
- * @param {object} formData All data currently in the form
- * @returns
+ * Adds a new `applicant` object to the start of the `formData.applicants`
+ * array. This is used to add the certifier info to the first applicant
+ * slot so users don't have to enter info twice if the certifier is also an app.
+ * @param {object} formData standard formData object
+ * @param {string} path formconfig page path of the page where this function should apply
+ * @param {object} name standard fullNameUI name to populate
+ * @param {string} email email address to populate
+ * @param {string} phone phone number to populate
+ * @param {object} address standard addressUI address object to populate
  */
-export function getNameKeyForSignature(formData) {
-  let nameKey;
-  if (formData.certifierRole === 'sponsor') {
-    nameKey = 'veteransFullName';
-  } else if (formData.certifierRole === 'applicant') {
-    nameKey = 'applicants[0].applicantName';
+export function populateFirstApplicant(
```

Comment:
This new function lets us prefill the first applicant in the list loop with the certifier's information if the conditions are right.